### PR TITLE
exp queue: Add two command `queue remove` and `queue kill`

### PR DESCRIPTION
**Pull request by karajan1001**
fix: #7591

1. Add a new command `dvc queue`
2. Add two sub-command `dvc queue remove` and `dvc queue kill`
3. Add a unit test to test them

* [x] ❗ I have followed the [Contributing to DVC](https://dvc.org/doc/user-guide/contributing/core) checklist.

* [ ] 📖 If this PR requires [documentation](https://dvc.org/doc) updates, I have created a separate PR (or issue, at least) in [dvc.org](https://github.com/iterative/dvc.org) and linked it here.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

wait for #7714

### CODE REVIEW

**Comment by karajan1001** [comment]
Current UI

```bash
$ dvc queue --help
usage: dvc queue [-h] [-q | -v] {remove,kill} ...

Commands to manage dvc task queue.
Documentation: <https://man.dvc.org/queue>

positional arguments:
  {remove,kill}  Use `dvc queue CMD --help` to display command-specific help.
    remove       Remove experiments in queue
    kill         Kill experiments in queue

optional arguments:
  -h, --help     show this help message and exit
  -q, --quiet    Be quiet.
  -v, --verbose  Be verbose.

$ dvc queue remove --help
usage: dvc queue remove [-h] [-q | -v] [--all] [<experiment> [<experiment> ...]]

Remove experiments in queue
Documentation: <https://man.dvc.org/queue/remove>

positional arguments:
  <experiment>   Experiments in queue to remove.

optional arguments:
  -h, --help     show this help message and exit
  -q, --quiet    Be quiet.
  -v, --verbose  Be verbose.
  --all          Remove all experiments in queue.

$ dvc queue kill --help
usage: dvc queue kill [-h] [-q | -v] [<experiment> [<experiment> ...]]

Kill experiments in queue
Documentation: <https://man.dvc.org/queue/kill>

positional arguments:
  <experiment>   Experiments in queue to kill.

optional arguments:
  -h, --help     show this help message and exit
  -q, --quiet    Be quiet.
  -v, --verbose  Be verbose.
```

**Comment by dberenbaum** [comment]
Should I wait for #7714 before testing this?

File: dvc/commands/queue/kill.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import logging
+
+from dvc.cli.command import CmdBase
+from dvc.cli.utils import append_doc_link
+
+logger = logging.getLogger(__name__)
+
+
+class CmdQueueKill(CmdBase):
+    """kill exp task in queue"""
```
Comment (pmrowla) [inline dvc/commands/queue/kill.py:11]:
```suggestion
    """Kill exp task in queue."""
```

File: dvc/commands/queue/remove.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+    def run(self):
+        if self.args.all:
+            removed_list = self.repo.experiments.celery_queue.clear()
+        else:
+            removed_list = self.repo.experiments.celery_queue.remove(
+                revs=self.args.experiment
+            )
+
+        removed = ",".join(removed_list)
```
Comment (pmrowla) [inline dvc/commands/queue/remove.py:22]:
```suggestion
        removed = ", ".join(removed_list)
```

File: dvc/commands/queue/remove.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+from dvc.cli.command import CmdBase
+from dvc.cli.utils import append_doc_link
+from dvc.ui import ui
+
+logger = logging.getLogger(__name__)
+
+
+class CmdQueueRemove(CmdBase):
+    """remove exp in queue"""
```
Comment (pmrowla) [inline dvc/commands/queue/remove.py:12]:
```suggestion
    """Remove exp in queue."""
```

File: dvc/commands/queue/__init__.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+    queue_subparsers = queue_parser.add_subparsers(
+        dest="cmd",
+        help="Use `dvc queue CMD --help` to display " "command-specific help.",
+    )
+
+    fix_subparsers(queue_subparsers)
+    for cmd in SUB_COMMANDS:
+        cmd.add_parser(queue_subparsers, parent_parser)
+    fix_plumbing_subparsers(queue_subparsers)
```
Comment (pmrowla) [inline dvc/commands/queue/__init__.py:35]:
```suggestion
```

This call shouldn't be needed since there's no subcommands we want to hide right now

**Review by pmrowla — CHANGES_REQUESTED** [review]
Just a couple minor things, LGTM otherwise

**Comment by pmrowla** [comment]
@karajan1001 there's a linting error now (unused import), just fix that and then squash your commits and this can be merged

**Comment by karajan1001** [comment]
Just wondered why always got some tests failed?

**Comment by pmrowla** [comment]
The `exp show` tests are flaky because they haven't been updated to properly work with the celery changes.

**Comment by karajan1001** [comment]
@dberenbaum Could you please give help on the product side?

**Comment by pmrowla** [comment]
I think it may be easier for us to just merge these PRs into the dev branch so that testing can be done w/all of the commands available, and then iterate/fix bugs from there

**Comment by dberenbaum** [comment]
> I think it may be easier for us to just merge these PRs into the dev branch so that testing can be done w/all of the commands available, and then iterate/fix bugs from there

How soon do you expect to merge? If it's in the next couple days, I will wait. Otherwise, I'll try to look at what I can now.

**Comment by karajan1001** [comment]
> > I think it may be easier for us to just merge these PRs into the dev branch so that testing can be done w/all of the commands available, and then iterate/fix bugs from there
> 
> How soon do you expect to merge? If it's in the next couple days, I will wait. Otherwise, I'll try to look at what I can now.

I think we can wait in the next couple of days.

**Pull request merged**